configure.ac

```diff
@@ -256,7 +256,9 @@ m4_ifdef([project_oshmem], [OSHMEM_CONFIGURE_OPTIONS])
 # Set up project specific AM_CONDITIONALs
 AS_IF([test "$enable_ompi" != "no"], [project_ompi_amc=true], [project_ompi_amc=false])
 m4_ifndef([project_ompi], [project_ompi_amc=false])
-AM_CONDITIONAL([PROJECT_OMPI], [test "$project_ompi_amc" = "true"])
+AM_CONDITIONAL([PROJECT_OMPI], [test "$PROJECT_OMPI_amc" = "true"])
```

Comment:
This change in case doesn't seem right -- the lower case version is defined on the three lines above.